serialization.rst

```diff
@@ -86,6 +99,13 @@ Examples
     }
   } => { 'foo' : 42, 'prot' : "wine" }
 
+  ThatArrayClass implements MongoDB\BSON\ArraySerializable {
+    private $elements => [ 1, 2, 3 ],
+    function bsonSerialize() : array {
+        return $this->elements;
+    }
+  } => [ 1, 2, 3 ]
```

Comment:
And I would also add an example with array_values(), as this is going to end up in the docs at some point.